server/models/payment.py

```
@@ -201,6 +213,155 @@ def to_dict(self):
+    confirmation_received = db.Column(db.Boolean, default=False)
+    confirmed_at = db.Column(db.DateTime(timezone=True), nullable=True)
+    raw_callback_data = db.Column(db.JSON, nullable=True)
+    created_at = db.Column(
+        db.DateTime(timezone=True),
+        default=lambda: datetime.now(timezone.utc),
+        nullable=False,
+    )
+    currency = db.Column(db.String(10), nullable=False, default="KES")
+
+    @validates("amount")
+    def validate_amount(self, key, amount):
```

Ruff (0.13.3)

Unused method argument: `key`

(ARG002)